site/docs/bazel-vision.md

```diff
@@ -5,40 +5,49 @@ title: Bazel Vision
 any project, of any size or complexity, with tooling that's easy to adopt and
-extend.</font>
+extend.</font></p>
 
 *   **Engineers can take build fundamentals for granted.** Software developers
     focus on the creative process of authoring code because the mechanical
     process of build and test is solved. When customizing the build system to
     support new languages or unique organizational needs, users focus on the
     aspects of extensibility that are unique to their use case, without having
     to reinvent the basic plumbing.
+
 *   **Engineers can easily contribute to any project.** A developer who wants to
     start working on a new project can simply clone the project and run the
     build. There's no need for local configuration - it just works. With
     cross-platform remote execution, they can work on any machine anywhere and
     fully test their changes against all platforms the project targets.
     Engineers can quickly configure the build for a new project or incrementally
     migrate an existing build.
+
 *   **Projects can scale to any size codebase, any size team.** Fast,
     incremental testing allows teams to fully validate every change before it is
     committed. This remains true even as repos grow, projects span multiple
     repos, and multiple languages are introduced. Infrastructure does not force
     developers to trade test coverage for build speed.
 
-**We believe Bazel has the potential to fulfill this vision.** Bazel was built
+---
+
+**We believe Bazel has the potential to fulfill this vision.**
+
+Bazel was built
 from the ground up to enable builds that are reproducible (a given set of inputs
 will always produce the same outputs) and portable (a build can be run on any
-machine without affecting the output). These characteristics support safe
-incrementality (rebuilding only changed inputs doesn't introduce the risk of
-corruption) and distributability (build actions are isolated and can be
-offloaded). By minimizing the work needed to do a correct build and
-parallelizing that work across multiple cores and remote systems, Bazel can make
-any build fast. Bazel's abstraction layer—instructions specific to languages,
-platforms, and toolchains implemented in a simple extensibility language —
-allows it to be easily applied to any context.
+machine without affecting the output).
+
+These characteristics support safe incrementality (rebuilding only changed
+inputs doesn't introduce the risk of corruption) and distributability (build
+actions are isolated and can be offloaded). By minimizing the work needed to do
+a correct build and parallelizing that work across multiple cores and remote
+systems, Bazel can make any build fast.
+
+Bazel's abstraction layer — instructions specific to languages, platforms, and
+toolchains implemented in a simple extensibility language — allows it to be
+easily applied to any context.
 
 ## Bazel Core Competencies
```

Comment:
For this and all other level2+ headers, should we use **Capitalize All Words** formatting or **Only first word**?

This and the the very next header disagree. And we have inconsistent use all throughout the docset. I'm happy to do a docset-wide refactoring if we can agree on the answer.

I suspect the answer is **Only first word**. But I have no authority to point to.